ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -945,4 +1051,15 @@ static void printTopics(double rul, double soc, double tmp)
   ROS_INFO_STREAM("min_rul: " << std::to_string(rul) <<
                   ", min_soc: " << std::to_string(soc) <<
                   ", max_tmp: " << std::to_string(tmp));
+}
+
+/*
+ * Prints a warning to the user should they attempt to inject a fault while
+ * the power system is disabled. Only prints once.
+ */
+static void printFaultDisabledWarning()
+{
+  ROS_WARN_STREAM_ONCE("Faults cannot be injected while the power system "
```

Comment:
I suggest that this not be ONCE, unless that would flood the terminal with warnings.

Reply:
Because of the way fault injection works (the relevant function is called every cycle), this warning would print multiple times every cycle (once per activated fault) if it wasn't using the ONCE stipulation. I can switch it to THROTTLE with an arbitrary delay instead, if that's preferred.

Reply:
Yes, a throttle (of say one minute) sounds good.  This way the user is aware that there's a useless fault injection on, in case they miss that first warning.